plugins/tiddlywiki/multiwikiserver/templates/manage-user.tid

```diff
@@ -30,39 +30,50 @@ title: $:/plugins/tiddlywiki/multiwikiserver/templates/manage-user
 			</div>
 			
 			<div class="user-profile-roles">
-				<h2>User Roles</h2>
+				<h2>User Role</h2>
 				<ul>
-					<$list filter="[<user-roles>jsonindexes[]]" variable="role-index">
-						<li>
-							<$text text={{{ [<user-roles>jsonextract<role-index>jsonget[role_name]] }}}/>
-						</li>
-					</$list>
+					<li>
+						<$text text={{{ [<user-role>jsonget[role_name]] }}}/>
+					</li>
 				</ul>
 			</div>
 		</div>
 	</div>
 
-	<!-- <$reveal type="match" state="is-current-user-profile" text="yes"> -->
+	<$list filter="[<user-is-admin>match[yes]] [<is-current-user-profile>match[yes]]">
```

Comment:
IMO you should use the conditional shortcut here: https://tiddlywiki.com/#Conditional%20Shortcut%20Syntax

If the list-widget is used as an IF construction, we use the `variable="ignore"` or `variable="null"` because by default list-widget modifies the `currentTiddler` variable, which may not be desired. -- Just a thought